UNITTESTS/stubs/ATHandler_stub.cpp

```diff
@@ -145,42 +137,34 @@ void ATHandler::set_file_handle(FileHandle *fh)
 
 bool ATHandler::find_urc_handler(const char *prefix)
 {
-    struct oob_t *oob = _oobs;
-    while (oob) {
-        if (strcmp(prefix, oob->prefix) == 0) {
-            return true;
-        }
-        oob = oob->next;
-    }
-
-    return false;
+    return ATHandler_stub::bool_value;
 }
 
 void ATHandler::set_urc_handler(const char *urc, mbed::Callback<void()> cb)
 {
-    if (!cb) {
-        remove_urc_handler(urc);
-        return;
-    }
-
-    if (find_urc_handler(urc)) {
-        return;
-    }
-
-    struct oob_t *oob = new struct oob_t;
-    size_t prefix_len = strlen(urc);
-    if (prefix_len > _oob_string_max_length) {
-        _oob_string_max_length = prefix_len;
-        if (_oob_string_max_length > _max_resp_length) {
-            _max_resp_length = _oob_string_max_length;
-        }
-    }
-
-    oob->prefix = urc;
-    oob->prefix_len = prefix_len;
-    oob->cb = cb;
-    oob->next = _oobs;
-    _oobs = oob;
+//    if (!cb) {
+//        remove_urc_handler(urc);
+//        return;
+//    }
+
+//    if (find_urc_handler(urc)) {
+//        return;
+//    }
+
+//    struct oob_t *oob = new struct oob_t;
+//    size_t prefix_len = strlen(urc);
+//    if (prefix_len > _oob_string_max_length) {
+//        _oob_string_max_length = prefix_len;
+//        if (_oob_string_max_length > _max_resp_length) {
+//            _max_resp_length = _oob_string_max_length;
+//        }
+//    }
+
+//    oob->prefix = urc;
+//    oob->prefix_len = prefix_len;
+//    oob->cb = cb;
+//    oob->next = _oobs;
+//    _oobs = oob;
```

Comment:
Why is there commented out code here? If it is not required it should be removed ?

Reply:
removed